server/initializers/migrations/0280-description-support.ts

```diff
@@ -0,0 +1,53 @@
+import * as Sequelize from 'sequelize'
+
+async function up (utils: {
+  transaction: Sequelize.Transaction,
+  queryInterface: Sequelize.QueryInterface,
+  sequelize: Sequelize.Sequelize,
+  db: any
+}): Promise<void> {
+  {
+    const data = {
+      type: Sequelize.STRING(1000),
+      allowNull: true,
+      defaultValue: null
+    }
+    await utils.queryInterface.changeColumn('video', 'support', data)
+  }
+
+  {
+    const data = {
+      type: Sequelize.STRING(1000),
+      allowNull: true,
+      defaultValue: null
+    }
+    await utils.queryInterface.changeColumn('videoChannel', 'support', data)
+  }
+
+  {
+    const data = {
+      type: Sequelize.STRING(1000),
+      allowNull: true,
+      defaultValue: null
+    }
+    await utils.queryInterface.changeColumn('videoChannel', 'description', data)
+  }
+
+  {
+    const data = {
+      type: Sequelize.STRING(1000),
+      allowNull: true,
+      defaultValue: false
```

Comment:
null

Reply:
ah yes, you're right

Reply:
I've also increased the video support side on the client side, thanks for pointing that out